kotlin/buffer/src/main/java/com/esri/arcgisruntime/sample/buffer/MainActivity.kt

```diff
@@ -0,0 +1,162 @@
+ * limitations under the License.
+ *
+ */
+package com.esri.arcgisruntime.sample.buffer
+
+import android.graphics.Color
+import android.os.Bundle
+import android.view.MotionEvent
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.esri.arcgisruntime.geometry.GeodeticCurveType
+import com.esri.arcgisruntime.geometry.GeometryEngine
+import com.esri.arcgisruntime.geometry.LinearUnit
+import com.esri.arcgisruntime.geometry.LinearUnitId
+import com.esri.arcgisruntime.geometry.SpatialReferences
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.Basemap
+import com.esri.arcgisruntime.mapping.view.DefaultMapViewOnTouchListener
+import com.esri.arcgisruntime.mapping.view.Graphic
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay
+import com.esri.arcgisruntime.symbology.SimpleFillSymbol
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol
+import com.esri.arcgisruntime.symbology.SimpleRenderer
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlin.math.roundToInt
+
+class MainActivity : AppCompatActivity() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
+
+    // create a map with the Basemap
+    mapView.map = ArcGISMap(SpatialReferences.getWebMercator()).apply {
+      basemap = Basemap.createTopographic()
+    }
+
+    // create a fill symbol for geodesic buffer polygons
+    val geodesicOutlineSymbol = SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.BLACK, 2F)
+    val geodesicBufferFillSymbol = SimpleFillSymbol(
+      SimpleFillSymbol.Style.SOLID, Color.GREEN,
+      geodesicOutlineSymbol
+    )
+
+    // create a graphics overlay to display geodesic polygons and set its renderer
+    val geodesicGraphicsOverlay = GraphicsOverlay().apply {
+      renderer = SimpleRenderer(geodesicBufferFillSymbol)
+      opacity = 0.5f
+    }
+
+    // create a fill symbol for planar buffer polygons
+    val planarOutlineSymbol = SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.BLACK, 2F)
+    val planarBufferFillSymbol = SimpleFillSymbol(
+      SimpleFillSymbol.Style.SOLID, Color.RED,
+      planarOutlineSymbol
+    )
+
+    // create a graphics overlay to display planar polygons and set its renderer
+    val planarGraphicsOverlay = GraphicsOverlay().apply {
+      renderer = SimpleRenderer(planarBufferFillSymbol)
+      opacity = 0.5f
+    }
+
+    // create a marker symbol for tap locations
+    val tapSymbol = SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CROSS, Color.WHITE, 14F)
+
+    // create a graphics overlay to display tap locations for buffers and set its renderer
+    val tapLocationsOverlay = GraphicsOverlay().apply {
+      renderer = SimpleRenderer(tapSymbol)
+    }
+
+    // add overlays to the mapView
+    mapView.graphicsOverlays.apply {
```

Comment:
I think we should use this instead of the apply, a bit neater:

`mapView.graphicsOverlays.addAll(listOf(geodesicGraphicsOverlay, planarGraphicsOverlay, tapLocationsOverlay))`

Reply:
this is definitely neater, I'm trying to avoid using apply too much now